src/vstart.sh

```diff
@@ -1232,5 +1232,17 @@ if [ "$CEPH_DIR" != "$PWD" ]; then
     echo "export CEPH_KEYRING=$keyring_fn"
 fi
 
+# convenience wrapper for running cephfs-shell out of build dir
+if [ $MDS -gt 0  -a -n "$CEPH_ROOT" ]; then
+	cat > $CEPH_BIN/vstart-cephfs-shell << EOF
+#!/bin/bash
+
+export LD_LIBRARY_PATH=$CEPH_LIB
+export PYTHONPATH=$PYBIND:$CEPH_LIB/cython_modules/lib.$CEPH_PY_VERSION_MAJOR
+exec $CEPH_ROOT/src/tools/cephfs/cephfs-shell
```

Comment:
Is the `cephfs-shell` not installed in `$CEPH_BIN`? Shouldn't it refer to that?

Reply:
It doesn't seem to be. Maybe I'm missing some cmake -D magic for it?

Reply:
It seems we just need an install directive in `src/tools/cephfs/CMakelists.txt`. @tchaikov do you remember why we didn't do that?

Reply:
cephfs-shell is installed using `setup.py`. so we are using `distutils_install_module()` cmake function to install it. `$CEPH_BIN` is  `${CMAKE_BINARY_DIR}/bin` which is in turn used for `CMAKE_RUNTIME_OUTPUT_DIRECTORY`. python scripts do not fall into the category of cmake "runtime target files". also, strictly speaking, we don't "install" any of files into `$CEPH_BIN`. the artifacts are installed into `${CMAKE_INSTALL_PREFIX}`. so, unless you run "make install", none of the scripts or executables will be installed into `${CMAKE_INSTALL_PREFIX}`.